nestjs/src/discord-servers/dto/discord-server.dto.ts

```diff
@@ -2,13 +2,16 @@ import { DiscordServer } from '@entities/discordServer';
 import { ApiProperty } from '@nestjs/swagger';
 
 export class DiscordServerDto {
-  constructor(discordServer: DiscordServer) {
+  constructor(discordServer: DiscordServer, fullInfo: boolean = false) {
     this.id = discordServer.id;
-    this.createdDate = discordServer.createdDate;
-    this.updatedDate = discordServer.updatedDate;
     this.name = discordServer.name;
-    this.gratitudeUrl = discordServer.gratitudeUrl;
-    this.mentorsChatUrl = discordServer.mentorsChatUrl;
+
+    if (fullInfo) {
```

Comment:
please create a separate DTO for case when we need to return only name (`DiscordServerLimitedDto`). This condition makes DTO not valid as most of the fields are not initialized.

Reply:
actually, just use https://github.com/rolling-scopes/rsschool-app/blob/master/nestjs/src/core/dto/id-name.dto.ts